src/utils/convert.js

```diff
@@ -495,15 +559,13 @@ const renderIndex = (legacyApp) => {
     AFTER_RESPONSES: getAfterResponses(legacyApp),
   };
 
-  return renderAuth(legacyApp)
```

Comment:
Authentication has been moved to its own separate file `authentication.js`, so we no longer need to call `renderAuth()` when rendering `index.js`.